### ISSUE TITLE
Grid line and x-axis label missing when xaxis.labelCount = 2

### ISSUE THREAD

**Issue by FitnessEffect** [danielgindi/Charts#2929]
This issue only occurs when xaxis.labelCount = 2. For any other values the grid and x-axis labels display as expected.

<img width="383" alt="screen shot 2017-10-30 at 2 38 52 pm" src="https://user-images.githubusercontent.com/17306637/32189357-e2db3152-bd80-11e7-9653-3f8e2a00cde2.png">

In order to view this issue please access the project on my github account:
1) https://github.com/FitnessEffect/WorkoutTracker
2) Login with the following credentials or create your own account:
Email: test@gmail.com Password: testtest 
3)To view the issue:  Menu button (top left) > Personal > Progress

Awesome framework. Keep up the great work!

Stefan

**Comment by liuxuan30** [comment]
I think this might be a naughty bug. Can you debug at `computeAxisValues()` to see what' the `axis.entries` it calculates?

**Comment by FitnessEffect** [comment]
It only calculates the first and third labels.
axis.entries = [0.0, 2.0] when it should equal [0.0, 1.0, 2.0, 3.0]

**Comment by liuxuan30** [comment]
that depends; computeAxisValues() has logic to calculate the interval and the entries. So I'm not sure if it's really a bug, since you just set label count = 2, and it gives you two. Did you set `forceLabelEnabled`? if not, the label count might be 2 or more, but if true, only 2 then.

You can override this methods to calculate your own as work around.

**Comment by FitnessEffect** [comment]
setting forceLabelEnabled to either true or false does not fix the issue. Do you recommend I override computeAxisValues() in order to solve this issue?

**Comment by liuxuan30** [comment]
yes, but I would suggest could you please take a look if this is a bug or not. If true, can you fix it and make a PR?
